src/components/user/UserProfileTabs.tsx

```diff
@@ -33,10 +36,17 @@ function a11yProps(index: any) {
   };
 }
 
-function UserProfile() {
+type Props = {
+  user?: User;
```

Comment:
why optional?

Reply:
It depends where you want to handle the user loading, time when it's null. I could also write !!user && <UserProfileTabs user={user} />
Any preference ?

Reply:
yeh think its better! Cause you force at least you to provide a user props to the component :) 
But it can User | null ?